src/GlobalStyles.js

```diff
@@ -0,0 +1,23 @@
+import { createGlobalStyle } from 'styled-components'
+
+
+export default createGlobalStyle`
+  @import url('https://fonts.googleapis.com/css2?family=Kaushan+Script&display=swap');
+  @import url('https://fonts.googleapis.com/css2?family=Marcellus&display=swap');
```

Comment:
> At this time we do not recommend using @import within cGS due to some issues with how browsers process @import via the CSSOM APIs. Instead it's best to place these in your core index.html file (generated or static) within a typical <style> tag.
Source: https://styled-components.com/docs/faqs#note-regarding-css-import-and-createglobalstyle

Try to use CSS `@font-face` ruleset here or use HTML `<link>` or `<style>` to add the fonts in the index.html